keyboards/mechwild/mercutio/keymaps/jonavin/keymap.c

```diff
@@ -246,12 +244,12 @@ bool process_record_keymap(uint16_t keycode, keyrecord_t *record) {
             led_t led_state = host_keyboard_led_state();
             oled_set_cursor(8,0);
             uint8_t wpm_count;
-            char wpm_str[10];
+            char wpm_str[4];
             wpm_count=get_current_wpm();
 
             if (wpm_count > 020) { // how wpm when > 20
-                 sprintf(wpm_str, " WPM: %03d", wpm_count);
-                oled_write(wpm_str, false);
+                oled_write_P(PSTR(" WPM: "), false);
+                oled_write(itoa(wpm_count, wpm_str, 10), false);
```

Comment:
have you considered `get_u8_str`? 

https://docs.qmk.fm/#/squeezing_avr?id=oled-tweaks

Reply:
This is resolved, also removed redundant stdio and variable after this change